CMSgov/onemac
# Consolidated svgs and style fixes

Endpoint: https://d1p8aq41s2b9jv.cloudfront.net

### Details

Quick fixes and consolidation of svg's to clean up.

### Changes

- Delete of redundant SVG's
- Alt text fix

### Test Plan

1. Check out hamburger menu logged in and out of state submitter.
2. Check to see if OneMac logo appears correctly in the top right of the menu.
